machine_configuration/README.md

```diff
@@ -87,6 +87,16 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
```

Comment:
Commit log should be:

machine_configuration: Update the Readme for QAT provisioning

Signed-off-by: Manish Regmi <manish.regmi@intel.com>